datastream-bigquery/src/main/java/com/linkedin/datastream/bigquery/BigqueryTransportProviderAdmin.java

```diff
@@ -118,4 +188,75 @@ public void dropDestination(Datastream datastream) {
     public Duration getRetention(Datastream datastream) {
         return Duration.ofSeconds(0);
     }
+
+    BigqueryDatastreamConfiguration getConfigurationFromDatastream(final Datastream datastream) {
+        final Map<String, String> metadata = datastream.getMetadata();
+        final String schemaRegistryLocation = metadata.getOrDefault(METADATA_SCHEMA_REGISTRY_LOCATION_KEY, _legacyDefaultSchemaRegistryUrl);
```

Comment:
Agreed. Right now we have the transport provider metadata options documented in the RFC doc, but it would be nice if these options were better documented by the code itself, something similar to API docs maybe.